server/src/main/java/org/candlepin/resteasy/JsonProvider.java

```diff
@@ -107,6 +124,11 @@ private void configureHateoasObjectMapper(ObjectMapper mapper, boolean indentJso
         }
 
         SimpleFilterProvider filterProvider = new SimpleFilterProvider();
+        filterProvider = filterProvider.addFilter("DTOFilter", new DynamicPropertyFilter());
+
+        // TODO: Once all DTOs are ported to the openapi spec, stop adding the following
+        //  filters: OldDTOFilter, ConsumerFilter, EntitlementFilter, OwnerFilter, GuestFilter.
+        filterProvider = filterProvider.addFilter("OldDTOFilter", new DynamicPropertyFilter());
```

Comment:
This is probably the lowest priority thing I've ever requested, but could we call this LegacyDTOFilter rather than Old? I don't know why, but "old" feels very rough to me.

Reply:
No problem, `LegacyDTOFilter` sounds good

Reply:
Done